docs/tools/hedgehog-engine/common/levels.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Nice! I'll copy over your binary and add it to the Releases page of HedgeDocs, alongside all the other ones.

Btw, you still need to add this page to the `mkdocs.yml` file, otherwise it will not exist in the website!

Reply:
right, lemme do that